src/pipelines/ct_org.py

```diff
@@ -0,0 +1,82 @@
+"""Preprocessing pipeline for CT-ORG dataset."""
+
+import os
+import re
+from dataclasses import asdict, dataclass, field
+from functools import partial
+from typing import Any
+
+import cv2
+import numpy as np
+
+from base.extractors import BaseImgIdExtractor, BaseLabelExtractor, BaseStudyIdExtractor
+from base.pipeline import BasePipeline, PipelineArgs
+from config import dataset_config
+from config.dataset_config import DatasetArgs
+from constants import MASK_FOLDER_NAME
+from steps import (
+    AddUmieIds,
+    ConvertNii2Png,
+    CopyMasks,
+    CreateBlankMasks,
+    CreateFileTree,
+    DeleteImgsWithNoAnnotations,
+    DeleteTempPng,
+    GetFilePaths,
+    RecolorMasks,
+)
+
+
+class ImgIdExtractor(BaseImgIdExtractor):
+    """Extractor for image IDs specific to the Knee Osteoarthritis dataset."""
+
+    def _extract(self, img_path: str) -> str:
+        """Extract image id from img path."""
+        return os.path.basename(img_path).split("_")[-1]
+
+
+class StudyIdExtractor(BaseStudyIdExtractor):
+    """Extractor for study IDs specific to the Knee Osteoarthritis dataset."""
+
+    def _extract(self, img_path: str) -> str:
+        """Extract study id from img path."""
+        study_id = os.path.basename(img_path).replace("-", "_").split("_")[-2]
+        return study_id
+
+
+@dataclass
+class CTORGPipeline(BasePipeline):
```

Comment:
It is good practice to use abbreviations as other words for readability reasons (Only first letter large) [(Source)](https://www.approxion.com/capital-offenses-how-to-handle-abbreviations-in-camelcase/). Please change everywhere this is used.
![image](https://github.com/user-attachments/assets/48c38016-5a78-4d40-89de-66e3501bbe73)

```suggestion
class CtOrgPipeline(BasePipeline):
```